### PR TITLE
built a Release your snap page for Snapcraft rebranding

### PR DESCRIPTION
**Pull request by chillkang**
## Done
- built a Release your snap page for Snapcraft rebranding

## How to QA
- Go to https://snapcraft-io-4248.demos.haus/about/release
- Compare designs to [web](https://app.zeplin.io/project/64107f86ec62b30f2acff2de/screen/641088808f9ea410a6dcb5f1) and [mobile](https://app.zeplin.io/project/64107f86ec62b30f2acff2de/screen/642177382c854822aeed884c)

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-2961

### CODE REVIEW

File: templates/about/release.html [hunk 1]
```diff
@@ -27,8 +27,58 @@ <h2 class="p-heading--4">
...
-  </div>
-  <div class="p-strip is-shallow">
+  </div> -->
+  <section class="p-strip--light u-no-padding--top" style="padding-bottom: 64px;">
+    <div class="u-fixed-width u-no-margin--bottom u-hide--small" style="padding-top: 64px; height: 0;">
+      <hr>
+    </div>
+    <div class="row">
+      <div class="col-6 u-hide--small">
+        <h1 class="p-heading--2">Releasing UI</h1>
```
Comment (Lukewh) [inline templates/about/release.html:37]:
```suggestion
        <h1 class="p-heading--2">Release UI</h1>
```

File: templates/about/release.html [hunk 2]
```diff
@@ -27,8 +27,58 @@ <h2 class="p-heading--4">
...
+      </div>
+      <div class="col-6 u-hide--medium u-hide--large" style="padding-top: 35px;">
+        <h1 class="p-heading--2">Releasing UI</h1>
```
Comment (Lukewh) [inline templates/about/release.html:40]:
```suggestion
        <h1 class="p-heading--2">Release UI</h1>
```

**Review by Lukewh — APPROVED** [review]
:+1: LGTM, just a copy change

File: templates/about/release.html (hunk 1 above)
Comment (chillkang) [reply templates/about/release.html]: @Lukewh I made a change :)

File: templates/about/release.html (hunk 2 above)
Comment (chillkang) [reply templates/about/release.html]: this one too!

**Review by steverydz — APPROVED** [review]
LGTM 👍

**Pull request merged**